### DISCUSSION TITLE
Rinkeby etherscan verification pending but never happens

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#762] [Q&A] [answered]
I am trying to upload my FundMe.sol contract to Etherscan.io but when I run `brownie run scripts/deploy.py --network rinkeby` on the terminal it waits for rinkeby etherscan to process and verify the contract.
But the problem is it gets stuck at the verification process
![Verification Pending](https://user-images.githubusercontent.com/70758518/149094539-2d716d0c-463b-4786-85ad-dae03bb2b9f7.png)

I kept it running and it has been 4 hours and its still pending.
any solutions to this would be nice

**Comment by cromewar** [comment, marked as answer]
Hello @InsidTrifl This could be due to the "duplication" of the contract, if your code is exactly the same as on the course, it'll be automatically verified. Take in mind the same group of characters will generate the exact bytecode, also make sure your private key from EtherScan is correctly added to your env variables. anyways try the following:

1. change a variable name, this will change the bytecode, and try again
2. Again be sure your private key for etherscan api y correctly added.
3. don't modify the code but set `verify=True` to false, and see if it gets automatically verified.

**Reply by ghost** [reply]
I did everything correctly and was still pending. I ran it again this morning and it works now. Must be something wrong on Etherscans end?

**Reply by cromewar** [reply]
Yep, could be indeed, I'm glad is solved now anyways.

Closing this thread now.
